src/main/java/com/intuit/graphql/orchestrator/schema/transform/FederationTransformerPostMerge.java

```diff
@@ -184,4 +196,16 @@ private void checkFederationFieldDirectives(EntityMergingContext entityMergingCo
       });
 
   }
+
+  private void checkFederationTypeDirectives(String typeName, List<Directive> typeDirectives, EntityMergingContext entityMergingContext) {
```

Comment:
should this be a FederationUtils?

Reply:
I debated it, but I I don't think there is a need since this portion shouldn't be called again by another class.